packages/frontend/src/features/metricsCatalog/components/MetricCatalogCategoryFormItem.tsx

```diff
@@ -132,15 +135,35 @@ export const MetricCatalogCategoryFormItem: FC<Props> = ({
     }
 
     return (
-        <Group spacing={4} position="apart" w="100%">
-            <CatalogCategory category={category} onClick={onClick} />
-            <ActionIcon
-                size="xs"
-                variant="subtle"
-                onClick={() => setIsEditing(true)}
-            >
-                <MantineIcon icon={IconDots} color="gray.6" size={14} />
-            </ActionIcon>
+        <Group
+            ref={hoverRef}
+            px={4}
+            py={3}
+            pos="relative"
+            position="apart"
+            sx={(theme) => ({
+                borderRadius: theme.radius.sm,
+                '&:hover': { backgroundColor: theme.colors.gray[2] },
+            })}
+        >
+            <UnstyledButton
+                onClick={onClick}
+                w="100%"
+                h="100%"
+                pos="absolute"
+            ></UnstyledButton>
+            <CatalogCategory category={category} onRemove={onDelete} />
```

Comment:
```suggestion
            <CatalogCategory category={category} />
```

so there's no `x` here next to the tag 
<img width="326" alt="image" src="https://github.com/user-attachments/assets/ba42e921-97f9-42fe-85d3-642443019d22">